CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [5.2.0](https://github.com/klassijs/klassi-js/compare/v5.0.0...v5.2.0) (2023-06-06)
+
+
+### Features
+
+* added new features and refactor s3Upload ([c53736d](https://github.com/klassijs/klassi-js/commit/c53736d5d9ae1c805cc9b985a9eb6657bf762fa5))
```

Comment:
As with the other PR, let's get rid of repetitions on the Changelog

Reply:
removed